README.md

```diff
@@ -1296,6 +1319,18 @@ Finally, let's implement the chat functionality. For this workshop, we're going
 
 #### Create `chat` Edge Function
 
+_Note: In the following steps, we show how to integrate with the
+ models provided by OpenAI. There are alternatives though.
+Whichever provider you choose, you can reuse the provided code
+as long as the provider offers an OpenAI-compatible API.
+Such providers are [together.ai](https://docs.together.ai/docs/openai-api-compatibility#nodejs), 
+[fireworks.ai](https://readme.fireworks.ai/docs/openai-compatibility), 
+[endpoints.anyscale.com](https://docs.endpoints.anyscale.com/examples/work-with-openai/) 
+or a local model served with [Ollama](https://github.com/ollama/ollama/blob/main/docs/openai.md#openai-javascript-library). 
+You can find more providers [here](https://sdk.vercel.ai/docs/guides).
+We will add the code for Ollama using Notes, 
+and we leave the rest as an exercise for the reader._
+
```

Comment:
```suggestion
_Note: In this tutorial we use models provided by OpenAI to implement the chat logic.
However since making this tutorial, many new LLM providers exist, such as:
- [together.ai](https://docs.together.ai/docs/openai-api-compatibility#nodejs)
- [fireworks.ai](https://readme.fireworks.ai/docs/openai-compatibility)
- [endpoints.anyscale.com](https://docs.endpoints.anyscale.com/examples/work-with-openai/) 
- [local models served with Ollama](https://github.com/ollama/ollama/blob/main/docs/openai.md#openai-javascript-library)

Whichever provider you choose, you can reuse the code below (that uses the OpenAI lib) as long as they offer an OpenAI-compatible API _(all of providers listed above do)_. We'll discuss how to do this in each step using Ollama, but the same logic applies to all providers
```